ui/src/views/image/RegisterOrUploadTemplate.vue

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
```suggestion
```

Reply:
@DaanHoogland 
I have committed most of the suggestions, thanks.

regarding the smoke test, currently in each test, there is a check if the vnf template is ready, so there is no intermitt failures.
I can 
(1) merge all tests into 1 test
(2) or, register vnf template at each test and delete after test.
(3) keep current code
which one do you prefer ?

Reply:
@DaanHoogland 
I just checked two other tests: test_vm_life_cycle and test_kubernetes_clusters, it looks they both use the same vm or k8s cluster in all tests. I understand it will save lots of time creating new vms/clusters for each test

Reply:
I understand but what I am saying is that if we do it should be created in setupClass, not in a test.

Reply:
I have refactored the tests. 
In each test, the first step is to check if vnf template is registered.